cmd/groups/main.go

```diff
@@ -175,6 +177,20 @@ func main() {
 	}
 	defer domainsHandler.Close()
 
+	callCfg := callback.Config{}
+	if err := env.Parse(&callCfg); err != nil {
+		logger.Error(err.Error())
```

Comment:
Consider adding some context to the error like "failed to parse callback config"